core/CMakeLists.txt

```diff
@@ -465,7 +466,7 @@ add_library (ZXing
 
 target_include_directories (ZXing
     PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>"
-    INTERFACE "$<INSTALL_INTERFACE:include>"
+    INTERFACE "$<INSTALL_INTERFACE:include/ZXing>"
```

Comment:
This will not work. See my recent change, doing exactly this and then reverting it after feedback from @vkrause .

Reply:
Reverted